pynxtools/dataconverter/readers/utils.py

```diff
@@ -41,6 +41,7 @@ class FlattenSettings():
     parent_key: str = "/ENTRY[entry]"
     sep: str = "/"
     is_in_section: bool = False
+    black_list: Optional[list] = None
```

Comment:
I'm thinking that we may should use a better name. Something like `ignore_keys`. What do you think?

Reply:
agreed